index.js

```diff
@@ -262,12 +262,13 @@ console.log(century)
  * Note that sucessfully invoking this function multiple times without refreshing your browser will continuously remove artists from the array until there are none left. If you refresh your browser, the data will reset.  
 */
 function removeArtist(arr, index) {
-    return arr.splice[index];
+    arr.splice(index, index);
+    return arr;
   }
   
-  removeArtist(artists, 0)
+removeArtist(artists, 0)
+console.log(removeArtist(artists, 0))
 
```

Comment:
{/* feedback
//make sure youre adding the correct thing into splice. one is the item to remove the other is an int aka a number or index
*/}